CHANGELOG.md

```diff
@@ -1,13 +1,22 @@
+### 1.5.6 (Dec 8, 2015)
+### general
+  - This release bundles a new version of JRuby - [1.7.23](http://jruby.org/2015/11/24/jruby-1-7-23.html), which fixes a memory leak issue reported on Windows when using the file input ([#3754](https://github.com/elastic/logstash/issues/3754)).
+
+### input
+  - File:
+    - Properly release file handles for older files which allows users to delete them ([#31](https://github.com/logstash-plugins/logstash-input-file/issues/31)).
```

Comment:
allows users to keep old files in the watched director without having to delete/rename them